### PR TITLE
Feature/navbar

### PR DESCRIPTION
**Pull request by blacksam07**
## Summary
- Add fixed navbar
- Add scroll to content
## Trello card
[31-navbar](https://trello.com/c/aKU5xzVz/31-navbar)

## Screenshots
image

navbar-scroll

- Screenshot updated 
image

### CODE REVIEW

File: scss/index.scss [hunk 1]
```diff
@@ -2,16 +2,17 @@
   display: flex;
   flex-flow: row wrap;
   justify-content: center;
-  width: 1000px;
+  padding: 75px 200px;
 }
 
 .list-item {
   box-shadow: 4px 4px 10px 0 $black-lighter-5;
```
Comment (betoobernal) [inline scss/index.scss:9]: the shadow is unlikely that be the same. I believe that is a little different. 🤔

File: scss/index.scss [hunk 2]
```diff
@@ -30,3 +31,41 @@
...
+  background: $white;
+}
+
+body::-webkit-scrollbar-thumb {
+  background: $white;
+}
+
+.navbar {
+  background-color: $white; 
+  box-shadow: 4px 4px 10px 0 $black-lighter-5;
```
Comment (betoobernal) [inline scss/index.scss]: the shadow is unlikely that be the same. I believe that is a little different. 🤔

File: index.html [hunk 3]
```diff
@@ -8,46 +8,55 @@
...
-    <a class="item-book-cover" href="detail.html"></a>
-    <span class="item-bold-text">Título del libro</span>
-    <span class="item-text">Autor del libro</span>
+<body>
+  <nav class="navbar">
+    <ul class="navbar-nav">
+      <li><a href="#" class="logo"></a></li>
+      <li><a href="#"><span>Logout</span></a></li>
+    </ul>
+  </nav>
```
Comment (betoobernal) [inline index.html]:
```suggestion
  <nav class="navbar">
      <a href="#" class="logo"></a>
      <a href="#"><span>Logout</span></a>
  </nav>
```
i think that it looks better thus. Okey, you should think now: "but... and the side spaces?" haha. Use the same styles that you used for the [container list](https://github.com/wolox-training/st-angular/blob/d36cdc7d09cd35f684a9d5c9316eb8c5f47e9122/scss/index.scss#L5)

Comment (betoobernal) [reply index.html]: I speak about `padding`

Comment (blacksam07) [reply index.html]: I can use `navbar ` and `navbar-nav` in a same tag and work fine

File: scss/index.scss [hunk 4]
```diff
@@ -2,16 +2,17 @@
...
+  padding: 75px 200px;
 }
 
 .list-item {
   box-shadow: 4px 4px 10px 0 $black-lighter-5;
+  flex: 1 0 20%;
   height: 300px;
   margin-top: 35px;
   margin-left: 35px;
+  max-width: 200px;
```
Comment (CrisVega08) [inline scss/index.scss:16]: add width: 100%;

File: scss/index.scss [hunk 5]
```diff
@@ -30,3 +31,40 @@
...
+  border-top: solid 5px $cerulean;
+}
+
+.navbar-nav {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
```
Comment (CrisVega08) [inline scss/index.scss]: sort properties

File: scss/index.scss [hunk 6]
```diff
@@ -30,3 +31,40 @@
...
+  justify-content: space-around;
+}
+
+.logo {
+  height: 70px;
+  width: 210px;
+  background: center / cover no-repeat url("resources/logo-wolox.png");
```
Comment (CrisVega08) [inline scss/index.scss]: sort properties

File: scss/index.scss [hunk 7]
```diff
@@ -30,3 +31,40 @@
...
+  background: $white;
+}
+
+.navbar {
+  background-color: $white; 
+  box-shadow: 0 4px 10px 0 $black-lighter-5;
+  height: 70px;
+  position: fixed;
+  width: -webkit-fill-available;
+  border-top: solid 5px $cerulean;
```
Comment (CrisVega08) [inline scss/index.scss]: sort properties

File: scss/index.scss [hunk 8]
```diff
@@ -2,16 +2,17 @@
   display: flex;
   flex-flow: row wrap;
   justify-content: center;
-  width: 1000px;
+  padding: 75px 200px;
 }
 
 .list-item {
   box-shadow: 4px 4px 10px 0 $black-lighter-5;
+  flex: 1 0 20%;
```
Comment (CrisVega08) [inline scss/index.scss]: why flex-basics 20%?

**Review by CrisVega08 — CHANGES_REQUESTED** [review]

**Comment by CrisVega08** [comment]
@blacksam07 check properties for logout text, I am seeing different to trello card

File: scss/index.scss (hunk 8 above)
Comment (blacksam07) [reply scss/index.scss]: Because I thought that if I used this percentage, the spaces would be distributed

**Review by CrisVega08 — APPROVED** [review]

**Pull request merged**